addons/police/CfgSerialKillers.hpp

```diff
@@ -0,0 +1,80 @@
+class CfgSerialKillers {
+    class Equipment {
+        class Vanilla {
+            class Common {
```

Comment:
`Common` does not feel right in `police` component, but spliting this into multiple components does not make it feel right either as it will be harder to maintain IMHO. Consider separate config only component? (`loadouts` or smth.)

Reply:
Yes, I thought about it and ended with new component being necessary to have everything much clearer and easier to maintain.